cli2/lib/cli.js

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env node
```

Comment:
Is this the JS that TS generates? I don't think we should add that to source control.

Reply:
These files still seem to be in your PR